socket/src/main/java/net/runelite/client/plugins/socket/plugins/specialcounterextended/SpecialCounterExtendedPlugin.java

```diff
@@ -1,11 +1,32 @@
-package net.runelite.client.plugins.specialcounterextended;
+/*
```

Comment:
this should be a completely different plugin, not bundled into the socket plugin.